x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -148,6 +149,11 @@ export class ObservabilityAIAssistantClient {
           title = params.title || '';
         }
 
+        let responseLanguage = 'English';
+        if ('responseLanguage' in params) {
+          responseLanguage = params.responseLanguage || 'English';
+        }
```

Comment:
```suggestion
        const responseLanguage = params.responseLanguage || 'English'
```

And then make `responseLanguage` optional in the `complete` function

Reply:
Fixed in https://github.com/elastic/kibana/pull/176444/commits/2730876eb46edb3e5e7547b847c782888370044a